unity_outline.md

```diff
@@ -0,0 +1,112 @@
+# Game Development with Unity
+
+> ####Useful Links for the course
+>
+> **Unity**
+>
+> - [Download Unity](http://unity3d.com/unity/download)
+> - [Unity Scripting API](http://docs.unity3d.com/ScriptReference/index.html)
+> - [Unity Video Tutorials](http://unity3d.com/learn/tutorials/modules/beginner/editor)
+> - [Unity Community Support](http://unity3d.com/community)
+
+## Foundation Phase
+
+In this phase students will build the two dimensional game Pong as well as slightly more challenging three dimensional game where they navigate a ball through a maze.  By the end of this foundation, the student should be equipped with knowledge pertaining to the Unity application, basic C# knowledge, and how Unity ties all pieces of a game together to make it a functional unit.  All assets will be provided for his phase.
+
+0. Setting up Our project, creating a githug profile and initializing a git repository
+    - Explain revision control and why we need it
+
+1. Brief introduction to the Unity Interface
+
+    What is Unity?
+    [Unity](http://en.wikipedia.org/wiki/Unity_(game_engine)) is a cross-platform game creation system developed by Unity Technologies, including a game engine and integrated development environment (IDE).  It is used to develop video games for web sites, desktop platforms, consoles, and mobile devices.
+
+    - What do different views in the software do? 
+        - Game
+        - Scene (explain here that this is effectively a level?)
+        - Hierarchy
+        - Inspector
+        - Project
+        - Console
+    - Teach the student that the views can be manipulated and dragged around to their liking.
+    - Explain the static 'Layers' and 'Layout' button at the top right of the software.
+
+2. Setting up the Environment
+    - Creating Your First Game Board: Most games start with an environment or level, in Pong we have a simple board.  In this checkpoint we show how to create the game board for Pong which is effectively creating a GameObject, naming it, setting the proper sprite, then setting the layer properties.
+
+3. Camera Setup
+    - Setting Up The Camera: In this checkpoint we show how to set the camera for Pong and why the camera needs to be set in a certain fashion.
+    - Setting Up the Boundaries for our Game Board: In this checkpoint we show how to add boundaries to the Pong game so that the players move within the boundary constraints.
+        - Breakpoint: Introduction to 'MonoDevelop' which is ehe IDE that Unity has chosen to use.
+            - Explain what an IDE is and why we use them
```

Comment:
@mikesjewett I'm completely cool with that.  I think the scope of topics we'd have to cover to teach C# would be quite a bit.  I pulled the idea of trying to teach the programming topics within the context of the project from this bit of the curriculum development guide, "Project oriented roadmaps work better because they tie new knowledge to very tangible features. It's a lot harder for a student to understand what MVC is in the abstract, but it's much easier for them to understand it in the context of creating a user sign up flow.".  So the idea was that if they could mentally map a programming concept to something tangible it would potentially ease their learning curve.